src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css

```diff
@@ -0,0 +1,58 @@
+.gameInfo {
+    position: relative;
+    top: -8px;
+    width: 91.18%;
+    padding-left: 4.41%;
+    padding-right: 4.41%;
+
+    .titleAndRanking {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
+
+        .ranking {
+            font-family: 'Exo';
+            font-style: normal;
+            font-weight: 700;
+            font-size: 18px;
```

Comment:
For font-sizes you should use rem as well, since it will make everything scalable